.drone.yml

```
@@ -183,7 +181,7 @@ steps:
     commands:
       - "curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash"
       - apt-get install -y git-lfs
-      - make test-mysql-migration integration-test-coverage
+      - make test-mysql-migration integration-test-coverage#TestAPIDeleteFile
```

```suggestion
      - make test-mysql-migration integration-test-coverage
```

Can you remove this here to see it succeeding? I'm marking it as a change request to prevent accidental merging.

---

There is no concern for mysql tests for this particular problem it won't show anything new and having it run all of the tests would just take away more unnecessary amd64 CI time while debugging.

The only thing that matters for now is how long it takes to write files on arm64 which is still way off unfortunately -- even just running just the one test it already appears to have hit some kind of limit or other issue that makes writing very slow.